src/declarativedbusinterface.cpp

```diff
@@ -426,6 +428,10 @@ void DeclarativeDBusInterface::pendingCallFinished(QDBusPendingCallWatcher *watc
 
     if (reply.isError()) {
         qmlInfo(this) << reply.error();
+        QJSValue errorCallback = callbacks.second;
+        if (errorCallback.isCallable()) {
+            errorCallback.call();
```

Comment:
It makes sense to add the error check to both callback.call() and errorCallback.call() in a separate commit.

I would assume that call() fails if there's a JavaScript exception when running the function. So a syntactically correct function that calls an non-existing function or perhaps does a divide by zero would do the trick.